src/test/java/gov/usgs/wma/waterdata/DiscreteGroundWaterRulesTest.java

```diff
@@ -0,0 +1,58 @@
+package gov.usgs.wma.waterdata;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DiscreteGroundWaterRulesTest {
+
+	DiscreteGroundWater dga;
+	DiscreteGroundWaterRules rules = new DiscreteGroundWaterRules();
+
+	@BeforeEach
+	void setUp() {
+		dga = new DiscreteGroundWater();
+	}
+
+	@Test
+	void recognizedFieldVisitCommentsShouldParseToCorrectDateTimeAccuracyCode() {
+
+		dga.setFieldVisitComments(DateTimeAccuracy.HOUR.getMatchString());
+
+		rules.apply(dga);
+
+		assertEquals(DateTimeAccuracy.HOUR.getCode(), dga.getDateTimeAccuracyCode());
+		assertEquals(DateTimeAccuracy.HOUR.getText(), dga.getDateTimeAccuracyText());
+	}
+
+	@Test
+	void unrecognizedFieldVisitCommentsShouldParseToCorrectDateTimeAccuracyCode() {
+		dga.setFieldVisitComments("XYZ");
+
+		rules.apply(dga);
+
+		assertEquals(DateTimeAccuracy.MINUTE.getCode(), dga.getDateTimeAccuracyCode());
+		assertEquals(DateTimeAccuracy.MINUTE.getText(), dga.getDateTimeAccuracyText());
+	}
+
+	@Test
+	void emptyFieldVisitCommentsShouldParseToMINUTEDateTimeAccuracyCode() {
+		dga.setFieldVisitComments("");
+
+		rules.apply(dga);
+
+		assertEquals(DateTimeAccuracy.MINUTE.getCode(), dga.getDateTimeAccuracyCode());
+		assertEquals(DateTimeAccuracy.MINUTE.getText(), dga.getDateTimeAccuracyText());
+	}
+
+	@Test
+	void nullFieldVisitCommentsShouldParseToMINUTEDateTimeAccuracyCode() {
+
+		dga.setFieldVisitComments(null);
+
+		rules.apply(dga);
+		assertEquals(DateTimeAccuracy.MINUTE.getCode(), dga.getDateTimeAccuracyCode());
+		assertEquals(DateTimeAccuracy.MINUTE.getText(), dga.getDateTimeAccuracyText());
+	}
+}
```

Comment:
Another nice part about business rules being in 'Java land' is that I can really unit test them completely - way less hassle then requiring db level tests for business rules.